### DISCUSSION TITLE
useMantineDefaultProps is not a function

### DISCUSSION THREAD

**Discussion by MuttakinHasib** [mantinedev/mantine#1083] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/44552983/159769604-9d5db84d-1baf-4204-a6cd-2ed9632d0d6d.png)
![image](https://user-images.githubusercontent.com/44552983/159769801-d8268387-29cb-446b-a79d-59f68a831ae7.png)

**Comment by rtivital** [comment, marked as answer]
You have mantine packages version mismatch, make sure all packages have the same version in package.json
